test/appium/tests/base_test_case.py

```diff
@@ -195,4 +195,12 @@ def teardown_class(cls):
 
 class MultipleDeviceTestCase(environments[pytest.config.getoption('env')]):
 
-    pass
+    def setup_method(self, method):
+        super(MultipleDeviceTestCase, self).setup_method(method)
+        self.senders = dict()
+
+    def teardown_method(self, method):
+        for user in self.senders:
+            api_requests.faucet(address=self.senders[user]['address'])
+        super(MultipleDeviceTestCase, self).teardown_method(method)
```

Comment:
Why do you add faucet requests for MultipleDeviceTestCase only? We have transactions tests with only one driver (SingleDeviceTestCase), I think it'd be useful to add it there also

Reply:
Agree. But it will require refactoring of quite a few tests. Let's do it in a separate PR. I added this as a note in 167 board, https://github.com/status-im/status-react/projects/17#card-9619942